app/messages/Messages.scala

```diff
@@ -545,6 +546,11 @@ object Messages {
       english = "Select yes if you are the account holder"
     )
 
+    // BARs
```

Comment:
Maybe put this in a object for the bars form, this file should only contain things of type Message(english: String, welsh: Option[String])

Reply:
ALthough I'm not that fussed on this, so feel free to ignore

Reply:
I'll think about where they can go...